test/test_ewald.jl

```diff
@@ -18,13 +18,14 @@
     positions = [[0,0,0]]
     cryst = Crystal(latvecs, positions)
     infos = [SpinInfo(1, S=1, g=1)]
-    sys = System(cryst, (1,1,1), infos, :dipole; units=Units.theory)
+    units = Sunny.PhysicalConsts(; μ0=1, μB=1)
```

Comment:
I wonder if this should be the default for theory units (permeability of 1) rather than simply disallowing dipole-dipole.

Reply:
As far as I know, the only current use case would be for testing, or for theoretical models designed specifically to study dipole-dipole effects. In the latter case, the theorist will likely want a tunable dipole-dipole interaction strength, not some specific arbitrary one.